src/App.jsx

```diff
@@ -1,9 +1,82 @@
+import { useState } from "react";
 import "./App.css";
 
 function App() {
+  const [firstName, setFirstName] = useState("");
+  const [lastName, setLastName] = useState("");
+  const handleFormSubmit = () => {
+    alert(`Hello ${firstName} ${lastName} !`);
+  };
```

Comment:
I think it is not really important to make this function this big, it might seem like it is not a big deal but you can try to put it on a single line! instead of making the block. This improve the readability of the codebase.
```suggestion
  const handleFormSubmit = () =>  alert(`Hello ${firstName} ${lastName} !`);
```